lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
yeah its a lack of proper abstraction on AR's end ... 
have seen driver 'feature' detection in gems that is why I recommend to keep not using a name-space known to be used with another gem if we're not loading up that gem at runtime.